bin/00_deploy_pypi_replay.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+BASE_DIR=/data/tier0
+COUCHDB_DOCKER_DIR=$BASE_DIR/dockerMount/CMSKubernetes.couchdb/docker/pypi/wmagent-couchdb
+DEPLOY_DIR=$BASE_DIR/WMAgent.venv3
+WMAGENT_SECRETS=$BASE_DIR/admin/WMAgent.secrets.replay
+CERT=/data/certs/robot-cert-cmst0.pem
+KEY=/data/certs/robot-key-cmst0.pem
+#SPEC_DIR=$BASE_DIR/admin/Specs
+
+WMAGENT_TAG=2.3.0
+COUCHDB_TAG=3.2.2
+TIER0_VERSION=3.1.5
+#TIER0_ARCH=slc7_amd64_gcc630
+#DEPLOY_TAG=HG2402a
+cd $BASE_DIR
+echo "Removing deploy directory"
+rm -rf $DEPLOY_DIR
+
+echo "Deploying new wmagent"
+echo "WMAgent version $WMAGENT_TAG"
+echo "Tier0 version $TIER0_VERSION"
+sleep 3
+./deploy-wmagent-venv.sh -t $WMAGENT_TAG -y -s
+
+cd $DEPLOY_DIR
+
+echo "Activating virtual environment"
+sleep 3
+source $DEPLOY_DIR/bin/activate
+
+echo "wiping t0ast"
+sleep 3
+$BASE_DIR/00_wipe_t0ast.sh 2>&1 > /dev/null
+
+### Temporary files for proper initialization of the agent ###
+### These should be updated and included properly ###
+#######################################################
+cp $BASE_DIR/init.sh $DEPLOY_DIR/init.sh
+cp $BASE_DIR/manage-common.sh $DEPLOY_DIR/bin/manage-common.sh
+cp $BASE_DIR/config.py $WMA_CONFIG_DIR/config.py           
+#######################################################
```

Comment:
I understand. This was simply added for testing purposes, especially since the init.sh and manage-common.sh files that come with the environment are functioning properly yet.